core/modules/system/system.admin.inc

```diff
@@ -1566,41 +1566,99 @@ function system_site_information_settings_submit($form, &$form_state) {
 function system_cron_settings($form, &$form_state) {
   global $base_url;
   $form['#config'] = 'system.core';
+  $config = config('system.core');
+
+  $form['last_run'] = array(
+    '#type' => 'help',
```

Comment:
This isn't help. Let's leave this type as `'#markup'`.

Reply:
Left as `#markup`.